packages/fx-core/src/plugins/resource/spfx/utils/constants.ts

```diff
@@ -69,3 +69,17 @@ export class DeployProgressMessage {
   static readonly CreateSPAppCatalog = "Create Sharepoint app catalog";
   static readonly UploadAndDeploy = "Upload and deploy SPFx package to your tenant app catalog";
 }
+
+export class ManifestTemplate {
+  static readonly LOCAL_CONTENT_URL =
```

Comment:
The template will be unified to one and will update later once that work is done.